BUILD.md

```diff
@@ -23,32 +13,71 @@ Waf should always be called from the ardupilot's root directory.
 Differently from the make-based build, with Waf there's a configure step
 to choose the board to be used (default is `sitl`):
 
+## Basic use ##
+
+There are several commands in the build system for advanced usages, but here we
+list some basic examples.
+
+### Build ArduCopter for only one board ###
```

Comment:
"only one board" sounds weird here. We always build for one board at a time. Also, I think this title is unnecessary.